OctoKit/Event.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+import RequestKit
+
+public extension Octokit {
+
+    /**
+     Fetches all recent Events on the server
```

Comment:
Provide a link to the Github Docs about events.

Note: that currently this only fetches the last 30 events. (as that's the default page size for the github api)
to get the rest of the events we'll need to read the headers sent back, and look for the next API endpoint to hit to get more data.

Perhaps we should expand this API to let users, fetch a certain number of events? or let them reach back to a certain date [this is the one I'm interested in]